submissions/kissik/port-sniffer/package-lock.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "port-sniffer",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "async": {
+      "version": "3.1.0",
+      "resolved": "https://registry.npmjs.org/async/-/async-3.1.0.tgz",
+      "integrity": "sha512-4vx/aaY6j/j3Lw3fbCHNWP0pPaTCew3F6F3hYyl/tHs/ndmV1q7NW9T5yuJ2XAGwdQrP+6Wu20x06U4APo/iQQ=="
+    },
+    "commander": {
+      "version": "3.0.2",
+      "resolved": "https://registry.npmjs.org/commander/-/commander-3.0.2.tgz",
+      "integrity": "sha512-Gar0ASD4BDyKC4hl4DwHqDrmvjoxWKZigVnAbn5H1owvm4CxCPdb0HQDehwNYMJpla5+M2tPmPARzhtYuwpHow=="
+    }
+  }
+}
```

Comment:
This file is an artifact from `npm`. As the repo uses `yarn` it is recommended to use `yarn` for your project. Just remove this file with `git rm ./path/to/file`.

It is also OK to commit `yarn.lock` after you run `yarn` under your project sub-directory.

Reply:
Thank you for the code review!)
well, sth went wrong (( 
it seems that i've done all the require changes but travis doesn't want to check it anyway

Reply:
@kissik Your changes are OK. The next step will be integration with CI so that travis wouldn't complain. But we need to [fix your repo structure](https://github.com/kottans/nodejs-2019-homeworks/pull/21#issuecomment-550947597) first.